examples/python_interface/README.md

```diff
@@ -0,0 +1,58 @@
+## Python Interface
+The Python interface has been simplified so that it is no longer necessary to copy in parameters.  An updated version of the webcam example from <a href="https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/wiki/Getting-started:-using-the-new-features-of-MIGraphX-0.2">MIGraphX examples v0.2</a> is shown below.
+
+A careful comparison with previous example shows it is no longer necessary to allocate the parameters on the GPU or to explicitly copy parameters to the GPU or results from the GPU.  The Python interface handles this by default.
```

Comment:
The Python interface doesn't handles this, rather the Python API sets `offload_copy=True` by default when compiling the model(see [`migraphx.compile`](https://rocmsoftwareplatform.github.io/AMDMIGraphX/doc/html/reference/py.html#migraphx.compile)) which will insert copies into the migraphx::program.